AsyncDisplayKit/Layout/ASLayoutElement.h

```diff
@@ -253,7 +253,6 @@ extern NSString * const ASLayoutElementStyleLayoutPositionProperty;
  * set on an ASImageNode to display at a size different from the underlying image size. 
```

Comment:
Can we add an `@warning calling the getter to this method when the size's width or height are relative will cause an assert.`